department-of-veterans-affairs/caseflow
# J hoang/appeals 57366

Resolves [APPEALS-57366](https://jira.devops.va.gov/browse/APPEALS-57366)

# Description
Restricted the date selector in the Disposition component for VHA HLR/SC decision reviews. The decision date will now range from the form receipt date to the current date.

## Acceptance Criteria
- [ ] As a developer, I need to ensure the date range is restricted on the frontend. 

## Testing Plan
1. Login as VHAADMIN
2. Go to VHA Decision Review Queue
3. Click on any in progress task
4. Fill out the disposition for each issue
5. Open the decision date selector and ensure that you can only select dates from the form receipt date to the current date.

## Storybook Story
*For Frontend (Presentation) Components*

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?
